Fix few broken links (#2377)
kubeflow/website: content/en/docs/aws/customizing-aws.md

### Customize Authentication

> If you use [configuration file]({{% config-uri-aws-standard %}}), you can consider to change the default password in the [configuration file]({{% config-uri-aws-standard %}}). The configuration file contains:

Reviewer: ```suggestion
If you use the [configuration file]({{% config-uri-aws-standard %}}), you can consider to change the default password in the [configuration file]({{% config-uri-aws-standard %}}). The configuration file contains:
```

    username: admin@kubeflow.org
```

> If you use [configuration file with cognito]({{% config-uri-aws-cognito %}}), please see [this section](/docs/aws/authentication)

Reviewer: ```suggestion
If you use the [configuration file with cognito]({{% config-uri-aws-cognito %}}), please see [this section](/docs/aws/authentication)
```

### Customize IAM Role for Pods

> Please see [this section](/docs/aws/iam-for-sa)

Reviewer: ```suggestion

Refer to the [Setting up AWS IAM role for Service Account](/docs/aws/iam-for-sa) guide.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)

> Please see [this section](/docs/aws/private-access)

Reviewer: ```suggestion

Check the [How to create private EKS clusters](/docs/aws/private-access) guide.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)

### Customize Logging

> Please see [this section](/docs/aws/logging)

Reviewer: ```suggestion

Go to the [Add logging support for Kubeflow](/docs/aws/logging) guide for more information.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)
